### ISSUE TITLE
[bug] Plotting using matlab, some functions are missing in plot_learning_curves.m

### ISSUE THREAD

**Issue by neklom** [davidsandberg/facenet#1039]
Hi, I tried to use this code to plot some curves for my results and I found that there are some functions that are missing like "Undefined function or variable 'combineStyles'".

Does anyone have a working code or had defined that function please ? and thanks !

**Comment by hsm4703** [comment]
I have the same problem with you.

**Comment by kifaw** [comment]
Same here, I hope someone could help us because I really need it, I would appreciate that.

**Comment by Zhengyu-Li** [comment]
It seems just to combine colors, markers and line styles. You can just choose one, like '*b', '-b'. If you have several training results to plot, just set a cell like lineStyles={'*b', 'xr'}.

**Comment by hsm4703** [comment]
can you tell me detail plz><

**Comment by neklom** [comment]
yes @Zhengyu-Li I know thanks but I'm not familiar with Matlab and I tried to change the code a little bit but didn't work, I hope you could help if possible and thanks !

**Comment by hyderit** [comment]
This `combineStyles(  )` function is not defined in MATLAB. An implementation of the function is not provided either. That's what is causing the error message.

**Comment by hyderit** [comment]
A simple way to fix this issue is to rewrite both line styles. Something like:
```matlab
lineStyles = {'b.', 'go', 'rx', 'c+', 'm*', 'ys', 'kd'};
lineStyles2 = {'b-', 'g-.', 'r--', 'c:', 'm--', 'y-', 'k-.'};
```

**Comment by hsm4703** [comment]
￼￼￼
![1566555330979](https://user-images.githubusercontent.com/42968054/63585604-1ce3b580-c5d2-11e9-8fc5-497b1df975b7.jpg)

**Comment by hsm4703** [comment]
@hyderit   
how can i  fix the code

**Comment by wencoast** [comment]
@hyderit Hello, do you meet one problem as follows?

> `Unrecognized function or variable 'readlogs'.`

I am really looking forward to your reply and others help.

**Comment by kifaw** [comment]
@wencoast This script is used to visualize the train and test plots. You can instead use the h5py files produced after the training is done to visualize the graphs using python instead of using matlab.

In train_softmax.py script you can find in both train() and validate() functions a stat dictionary that contains information (Accuracies, losses, ...) about the train/test and validation of the model after it ends the training.
